interface/src/app/analyze/analysis/analysis.js

```diff
@@ -38,12 +48,7 @@ function AnalysisCtrl($scope, $log, $q, $state, SampleBin) {
       {'typename': 'medium'},
       {'typename': 'temperature'},
       {'typename': 'treatment'}
-    ],
-    updateMlModel: function(value) {
-      if (value !== null && value.id !== null) {
-        SampleBin.getActivityForSampleList(value.id);
-      }
-    }
+    ]
```

Comment:
The diff does not tell me anything I don't already know, so I can't yet see what's causing those errors. I'm still investigating. Pulling up a copy of @dongbohu's fork so I can try to reproduce here.